contributors/design-proposals/raw-block-pv.md

```diff
@@ -0,0 +1,571 @@
+# Local Raw Block Consumption via Persistent Volume Source
+
+Authors: erinboyd@, screeley44@, mtanino@
+
+This document presents a proposal for managing raw block storage in Kubernetes using the persistent volume source API as a consistent model of consumption.
+
+# Goals
+* Enable durable access to block storage
+* Support storage requirements for all workloads supported by Kubernetes
+* Provide flexibility for users/vendors to utilize various types of storage devices
+* Agree on API changes for block
+* Provide a consistent security model for block devices 
+* Provide block storage usage isolation
+* Provide a means for running containerized block storage offerings as non-privileged container
+
+# Non Goals
+* Support all storage devices natively in upstream Kubernetes. Non-standard storage devices are expected to be managed using extension
+  mechanisms.
+   
+# Value add to Kubernetes
+
+  Before the advent of storage plugins, emptyDir and hostPath were widely used to quickly prototype stateless applications in Kube. 
+  Both have limitations for their use in application that need to store persistent data or state. 
+  EmptyDir, though quick and easy to use, provides no real guarantee of persistence for suitable amount of time. 
+  Appropriately used as scratch space, it does not have the HostPath, became an initial offering for local storage, but had many
+  drawbacks. Without having the ability to guarantee space & ensure ownership, one would lose data once a node was rescheduled. 
+  Therefore, the risk outweighed the reward when trying to  leverage the power of local storage needed for stateful applications like 
+  databases.
+    
```

Comment:
There's a 'system utility' & bootstrap use case that should be noted.  If a fs health check, partition/format or other disk utility needs to run on the node it will want raw block.  Examples may be local storage partition & format of a raw disk, deploy kube to a bare metal node, utility pods that check disk health after reboot etc..